<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeRegions</class>
 <widget class="QMainWindow" name="TreeRegions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_NewRoot"/>
   <addaction name="action_New"/>
   <addaction name="separator"/>
   <addaction name="action_Delete"/>
   <addaction name="separator"/>
   <addaction name="action_Edit"/>
   <addaction name="separator"/>
   <addaction name="action_Yes"/>
   <addaction name="action_No"/>
  </widget>
  <action name="action_NewRoot">
   <property name="icon">
    <iconset resource="../resources/ViewWindowRes.qrc">
     <normaloff>:/images/root.png</normaloff>:/images/root.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Новый корневой</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="icon">
    <iconset resource="../resources/ViewWindowRes.qrc">
     <normaloff>:/images/trash.png</normaloff>:/images/trash.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="icon">
    <iconset resource="../resources/ViewWindowRes.qrc">
     <normaloff>:/images/note.png</normaloff>:/images/note.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="toolTip">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_Yes">
   <property name="icon">
    <iconset resource="../resources/ViewWindowRes.qrc">
     <normaloff>:/images/check.png</normaloff>:/images/check.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Yes</string>
   </property>
   <property name="toolTip">
    <string>Применить</string>
   </property>
  </action>
  <action name="action_No">
   <property name="icon">
    <iconset resource="../resources/ViewWindowRes.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;No</string>
   </property>
   <property name="toolTip">
    <string>Отменить</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="../resources/ViewWindowRes.qrc">
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Новый</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/ViewWindowRes.qrc"/>
 </resources>
 <connections/>
</ui>
